app/scripts/models/auth_brokers/base.js

```diff
@@ -50,10 +50,26 @@ define(function (require, exports, module) {
         // error if the browser is not configured to accept WebChannel messages
         // from this FxA server, which often happens when testing against
         // non-production servers. See #5114
-        this.setCapability('fxaStatus', this._notificationChannel.isFxaStatusSupported());
+        const isFxaStatusSupported = this._notificationChannel.isFxaStatusSupported();
+        this.setCapability('fxaStatus', isFxaStatusSupported);
+
+        if (isFxaStatusSupported) {
+          this.on('fxa_status', (response) => this.onFxaStatus(response));
+        }
+
       }
     },
 
+    /**
+     * Handle a response to the `fxa_status` message.
+     *
+     * @param {any} [response={}]
+     * @private
+     */
+    onFxaStatus (response = {}) {
+      this.setCapability('supportsPairing', response.capabilities && response.capabilities.pairing);
```

Comment:
I don't know if we have any kind of naming convention for capability strings; other examples seem to include "chooseWhatToSyncWebV1", "fxaStatus", and "allowUidChange".  In the absence of any unifying naming theme, "supportsPairing" seems fine 👍 .

Reply:
we can also try `allowPairing` to fit `allowUidChange`

Reply:
You have my 100% trust to pick a good name here, I was mostly sanity-checking that this wasn't wildly different to what we already had :-)